coderd/coderdtest/oidctest/testidp/main.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
This could be moved to `cmd/testidp`. I think it would be confusing to have packages that can be built sprinkled within the source tree.

Reply:
Good call :+1: